src/components/team/add-member-details.js

```diff
@@ -351,6 +356,21 @@ class AddMemberDetails extends Component {
                 ? 'Add Member Details'
                 : 'Modify Member Details'}
             </Header>
+            {( errorHandle || errorShortBio) && (
+              <Message error>
+                <Message.Header>There was some errors with your submission</Message.Header>
+                    {
+                      Object.keys(errorMessage).forEach(function(field) {
+                        const errors = errorMessage[field];
+                        errors.forEach(err =>
+                          (
+                            <p> {field} : {err} </p>
```

Comment:
@algomaster99 I see the control is reaching here but this line is not returned to the screen.

Reply:
@SuyashSalampuria `forEach` always returns `undefined`, which is the reason you cannot see any `<p>` element. You can use `map` and this will be resolved. :)